Classes/AssetExtraction/IngestAttachmentAssetExtractor.php

```diff
@@ -126,7 +132,7 @@ protected function getAssetContent(AssetInterface $asset): string
         }
 
         stream_filter_append($stream, 'convert.base64-encode');
-        $result = stream_get_contents($stream);
+        $result = stream_get_contents($stream, (int)$this->indexedChars);
```

Comment:
Does that work reliably? Binary content, just split at an arbitrary position is indexed correctly? I'd have thought that breaks…

Reply:
Good point. Currently testing in production. 
Alternatively there should be an option to exclude files > <givenFileSize> entirely

Reply:
Yep. Is borken.